### PR TITLE
Added Contributor Resources section

### PR DESCRIPTION
**Pull request by xomiamoore**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -20,17 +21,24 @@ You can also browse by Awesome Lists created by the community.
...
+     - **Browse our existing projects** and issues and contribute.
 
 We’ve put together some commonly asked [questions and answers here.](https://github.com/camunda-community-hub/community) 
 
+### Contributor resources & documentation
+* [Contributor guide](CONTRIBUTING.MD)
+* [Issue triage and labeling](issue-triage.md)
+* [Maintainer & reviewer expectations](#maintainer-reviewer-expectations.md)
+* [Automated release documentation](https://github.com/camunda-community-hub/community/blob/main/RELEASE.MD)
+* [Using our groupId for Java builds](groupId.md)
```
Comment (akeller) [inline profile/README.md]: I'm skeptical that these links are going to behave as expected. Can you double-check?

File: profile/README.md [hunk 2]
```diff
@@ -20,17 +21,24 @@ You can also browse by Awesome Lists created by the community.
...
+
 ## Troubleshooting
 
 If you have further questions, you can also ask via our [forum](https://forum.camunda.io/), the Camunda Community Hub [GitHub Discussions](https://github.com/camunda-community-hub/community/discussions), or reach out to the [Camunda DevRel team](https://github.com/orgs/camunda-community-hub/teams/devrel) via email: community@camunda.com.
```
Comment (akeller) [inline profile/README.md:44]: I know GitHub Discussions was on the list for Rin to explore, but what are your thoughts on it @xomiamoore?

Comment (xomiamoore) [reply profile/README.md:44]: I've done some digging, and I think it's a great idea to use GitHub discussions vs. the forums -- keep as much Camunda Community Hub discussion on here as possible. We can talk more about it in our 1:1 :)

**Review by xomiamoore — COMMENTED** [review]
~~Updated broken links~~ Ignore this, I'm fighting with VS Code 😓

**Comment by xomiamoore** [comment]
Ignore this, I'm fighting with VS Code 😓

**Review by akeller — APPROVED** [review]
No obvious issues spotted! Looks good!

**Pull request merged**